sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/implementation/faultinjection/FaultInjectionServerErrorResultInternal.java

```diff
@@ -117,7 +117,7 @@ public CosmosException getInjectedServerError(RxDocumentServiceRequest request)
 
             case READ_SESSION_NOT_AVAILABLE:
 
-                final String badSessionToken = "1:1#1#1=1#1=1";
+                final String badSessionToken = partitionKeyRangeId + ":" + "1#1#1=1#1=1";
```

Comment:
Makes more sense to use `partitionKeyRangeId` in the request when constructing the session token to prevent side-effects.